src/coreclr/jit/llvm.cpp

```diff
@@ -1223,6 +1271,81 @@ void Llvm::fillPhis()
     }
 }
 
+void Llvm::buildUnaryOperation(GenTree* node, Value* op1)
```

Comment:
This also needs to use `consumeValue` (and `op1` moved into the body like the others).